xds/internal/balancer/cdsbalancer/logging.go

```diff
@@ -20,10 +20,14 @@ package cdsbalancer
 
 import (
 	"fmt"
+
+	"google.golang.org/grpc/grpclog"
 )
 
 const prefix = "[cds-lb %p] "
 
+var logger = grpclog.Component("xds")
```

Comment:
What if we did this instead:

```
const prefix = "[cds-lb %p] "

var logger = grpclog.Component("xds")

func prefixLogger(p *cdsBalancer) *grpclog.PrefixLogger {
  return grpclog.NewPrefixLogger(logger, fmt.Sprintf("prefix, p))
}
```

And the `cdsBalancer.Build` would do `b.logger = prefixLogger(b)`

The same applies to other packages as well.

Reply:
Good call. Done!